firefox_puppeteer/ui/toolbars.py

```diff
@@ -347,22 +347,42 @@ class IdentityPopup(BaseLib):
 
     @property
     def box(self):
+        """ Provides access to the identity-box element.
```

Comment:
My understanding is that you'd like to remove the text outside of the returns statement, since it is redundant here. That's what I've done.